framework/src/base/ComputeResidualThread.C

```diff
@@ -101,7 +80,7 @@ ComputeResidualThread::onElement(const Elem *elem)
   _fe_problem.reinitMaterials(_subdomain, _tid);
 
 
-  const MooseObjectWarehouse<KernelBase> * storage;
+  MooseObjectWarehouse<KernelBase> const * storage;
```

Comment:
I'd be willing to bet that this is the only declaration in MOOSE using this order:

While your reasoning to change this makes sense, I think years of engrained practice go against this theory. Are you proposing that we declare none pointer types like this too (`int const i`)?

My specific problem with this order is that it's easy to confuse with the const pointer declaration:
These are not the same:
```
Foo * const p;   // const pointer
Foo const * p;   // const data (looks almost the same at first glance)

const Foo * p;  // more traditional const data (looks different from const pointer declaration)
```